test/README.md

```diff
@@ -0,0 +1,30 @@
+# Test
```

Comment:
Shouldn't we talk about "Integration tests" explicitely? We might also consider naming the "test" folder differently.
"Test" suggests that here all tests are located which is not true since tests are spread all over the "pkg" folder.